full_bubble_model.py

```diff
@@ -610,7 +627,22 @@ def _f(t, x, P_amb, alfa_M, T_inf, surfactant, P_v, mu_L, rho_L, c_L, ex_args, e
 
 """________________________________Solving________________________________"""
 
-def solve(cpar, t_int=np.array([0.0, 1.0]), LSODA_timeout=30.0, Radau_timeout=300.0, extra_dims=0, print_errors=False):
+# error codes description
```

Comment:
Error codes remain similar, however no solver errors are fatal. Some kind of solution is always retained. In case, when both solver fails, `de.solve()` will choose the one which failed at a later point in time. The second to last time step is usually used as final to avoid `nan `and `inf`.